nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -92,6 +93,39 @@
         .defaultValue("true")
         .required(true)
         .build();
+    public static final PropertyDescriptor PROXY_HOST = new PropertyDescriptor.Builder()
```

Comment:
@pvillard31 I saw that and considered it but ended up adopting the current approach due to my view that I prefer not supporting SOCKS ... Don't get me wrong, I appreciate the reason why SOCKS exists but most processors are only able to support HTTP so I rather provide... consistency (I know I have been sounding repetitive lately... 😄 )

Since FTP already supports SOCKS I reckon we can leave it there, but for SFTP I rather not add it at all.

Let me know if you agree and I will adjust accordingly.